L1Trigger/TrackFindingTracklet/src/MatchEngineUnit.cc

```diff
@@ -175,22 +168,18 @@ void MatchEngineUnit::reset() {
   idle_ = true;
   istub_ = 0;
   goodpair_ = false;
-  goodpair__ = false;
   havepair_ = false;
-  havepair__ = false;
 }
 
 int MatchEngineUnit::TCID() const {
   if (!empty()) {
     return peek().first->TCID();
   }
 
-  if (idle_ && !havepair_ && !havepair__) {
+  if (idle_ && !havepair_) {
     return 16383;
```

Comment:
Eliminate/justify magic number 16383. It's pow(2,14)-1. But where does 14 come from?